src/cdk/drag-drop/directives/drag.ts

```diff
@@ -330,9 +330,11 @@ export class CdkDrag<T = any> implements AfterViewInit, OnChanges, OnDestroy {
 
   /** Syncs the root element with the `DragRef`. */
   private _updateRootElement() {
-    const element = this.element.nativeElement;
+    const element = this.element.nativeElement as HTMLElement;
+    // Comment tag doesn't have closest method, so use parent's one.
+    const closestFn = element.closest || element.parentElement?.closest;
     const rootElement = this.rootElementSelector ?
-      element.closest<HTMLElement>(this.rootElementSelector) : element;
+      closestFn<HTMLElement>(this.rootElementSelector) : element;
```

Comment:
I think that this may break, because the `this` context isn't preserved like in the previous call.